src/storage/src/hummock/event_handler/uploader/mod.rs

```diff
@@ -727,7 +758,7 @@ struct UnsyncData {
     instance_table_id: HashMap<LocalInstanceId, TableId>,
     // TODO: this is only used in spill to get existing epochs and can be removed
     // when we support spill not based on epoch
-    epochs: BTreeMap<HummockEpoch, ()>,
+    epochs: BTreeMap<HummockEpoch, HashSet<TableId>>,
```

Comment:
Seems to be conflicting with https://github.com/risingwavelabs/risingwave/pull/17539